Automattic/jetpack
# Style updates for the first step in the disconnect dialog

#### Changes proposed in this Pull Request:

* This PR makes some small style improvements to the Jetpack disconnection flow.
* Numbers that are shown on the first screen of the disconnect flow should now be properly formatted/ localized.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:






* Check out this PR on your Jetpack test site
* If Jetpack is not already connected, complete the connection process. 
* Navigate to the Jetpack Dashboard
* In the "Connections" section of the page, click on the "Manage Connection" link to open the disconnect flow.
* Note that in the stat cards that show information about Jetpack on the site (like site stats or brute force attempts blocked) that the stat number should be centered vertically with the text. You can add more text using the web inspector to confirm this:
![Screen Shot 2021-12-23 at 4 06 55 PM](https://user-images.githubusercontent.com/18016357/147292337-61cad918-dbec-40fa-8786-4b891c9e2f3a.png)

* If the site has 2 or fewer stat cards, then some general benefits of Jetpack will show at the bottom of this first screen of the disconnection flow (this can be easiest to observe with a brand new site). These items should now also be styled using the same component that is used for the stats cards for consistent appearance: 
![Screen Shot 2021-12-23 at 4 04 57 PM](https://user-images.githubusercontent.com/18016357/147292371-0b2bbf61-fdbe-4674-b2d6-4fee4eb61996.png)
